tests/templates/pr-create.md.ejs.js

```diff
@@ -7,6 +7,8 @@
 
 "use strict";
 
+/* global describe, it */
```

Comment:
Oh, I think those are added via an `.eslintrc` file in the `tests/lib` directory. We can probably move that file up to be under `tests/` instead, or else, copy it to `tests/templates`